examples/smart_contracts/v1/javascript/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Algrorand",
+    "version": "0.1.0",
+    "scripts": {
+        "start": ""
+    },
+    "keywords": [],
+    "author": "Algorand",
+    "contributors": [
+        {
+            "name": "Edward D. Gaudio",
+            "email": "edwardgaudio@gmail.com"
+        }
+    ],
+    "license": "MIT",
+    "dependencies": {
+        "algosdk": "^1.9.1",
+        "dotenv": "^8.1.0"
+    }
```

Comment:
You'all probably want to get in here and standardize what this is. MIT license so all bases are covered, but you might want to adjust this.

Reply:
Perhaps we can omit the "contributors" section, as this is precedence [js-algorand-sdk](https://github.com/algorand/js-algorand-sdk/blob/develop/package.json)